readthedocs/subscriptions/querysets.py

```diff
@@ -0,0 +1,26 @@
+"""
+Querysets for djstripe models.
+
+Since djstripe is a third-party app,
+these are injected at runtime at readthedocs/subscriptions/apps.py.
```

Comment:
this can also be a collection of helpers, but using a queryset we don't need to import those and just use the model directly :)